### DISCUSSION TITLE
0.27.0版启用pyncmd后报错errno: -104

### DISCUSSION THREAD

**Discussion by gavin8282** [1715173329/UnblockNeteaseMusic#887] [Q&A]
服务端：
系统：Ubuntu 22.04.1 LTS
运行方式：MIN_BR=320000 NO_CACHE=true ENABLE_LOCAL_VIP=true /usr/bin/unblockneteasemusic-linux-arm64 -o pyncmd kuwo kugou -p 6200:6201 -s >>/tmp/mp3.log 2>&1 &
错误输出：    songName: "星河万里（航天特别版）"
    url: "http://cw.sycdn.kuwo.cn/5544d4c907e2206bc6ea1905ef833588/634a9a6c/resource/n3/21/13/4242913872.mp3"
ERROR: (provider/match) read ECONNRESET
    Error: read ECONNRESET
        at TLSWrap.onStreamRead (node:internal/stream_base_commons:217:20)
    errno: -104
    code: ECONNRESET
    syscall: read
客户端：
MacOS 12.6 云音乐版本2.3.8 (878)

**Comment by 1715173329** [comment]
？

**Comment by gavin8282** [comment]
> 

刚刚想换行，结果按了command+回车发布了

**Comment by 1715173329** [comment]
据说 vercel 被墙了，请检查你的网络环境。
